microscope_info.py

```diff
@@ -0,0 +1,66 @@
+import json
```

Comment:
This file could be placed inside it's own module (folder), just like `azure_storage_api.py`.

Reply:
Done